third_party/xla/xla/stream_executor/rocm/hipblaslt_wrapper.h

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 #define XLA_STREAM_EXECUTOR_ROCM_HIPBLASLT_WRAPPER_H_
 
 #define __HIP_DISABLE_CPP_FUNCTIONS__
+#define LEGACY_HIPBLAS_DIRECT
```

Comment:
Don't use this. Just include hipblas.h header.

Reply:
ok I will try it out, the point is the error about missing hipblas-common library only pops up when one is trying to use the newest version of hipblaslt..

Reply:
I have checked new hipblaslt header files, it seems that the compile error occurs when we are trying to build with new hipblaslt headers while hipblas-common library is not installed:

```
#include "hipblaslt/hipblaslt-export.h"
#include "hipblaslt/hipblaslt-version.h"
#ifndef LEGACY_HIPBLAS_DIRECT
#include <hipblas-common/hipblas-common.h>
#else
#include <hipblas/hipblas.h>
#endif
```

Hmm but actually this should not be the case .. only if hipblaslt is not proprely installed in the container. I think we can remove it then